### ISSUE TITLE
Removal DuckDuckGO (DDG)

### ISSUE THREAD

**Issue by cyberboh** [Divested-Mobile/Mull-Fenix#64]
Had security and privacy issue right now
`https://techcrunch.com/2022/05/24/ddg-microsoft-tracking-blocking-limit/`

As alternative, we can use `Mojeek` `https://www.mojeek.com/`

**Comment by Morgan-le-Fay** [comment]
Do you not read past the headline of articles? No wonder the internet is full of misinformation.
From the article you linked :
_Late yesterday, the researcher in question, Zach Edwards, [tweeted](https://twitter.com/thezedwards/status/1528808759027331072) the findings of his audit — saying he had found DDG’s mobile browsers do not block advertising requests made by Microsoft scripts on non-Microsoft web properties. (NB: This is a separate matter to what happens if you actually click on an ad when using DDG — as its [privacy policy](https://help.duckduckgo.com/duckduckgo-help-pages/company/ads-by-microsoft-on-duckduckgo-private-search/#:~:text=At%20that%20point%2C%20Microsoft%20Advertising%20will%20use%20your%20full%20IP%20address%20and%20user%2Dagent%20string%20so%20that%20it%20can%20properly%20process%20the%20ad%20click%20and%20charge%20the%20advertiser) clearly discloses all privacy bets are off at that point.)_

This is coming directly from their CEO : https://old.reddit.com/r/technology/comments/uxiah9/duckduckgo_caught_giving_microsoft_permission_for/i9xxjsn/

TL;DR: It's about their mobile browsers, not their search engine.
